packages/components/src/Button/button.tsx

```diff
@@ -15,15 +15,16 @@ export type ButtonProps = ButtonHTMLElementProps & {
   variant?: ClickableProps<"button">["variant"];
 };
 
-function Button({
-  as = "button",
-  variant = "flat",
-  color = "brand",
-  disabled = false,
-  type = "button",
-  size = "medium",
-  ...rest
-}: ButtonProps) {
+const Button = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) => {
+  const {
+    as = "button",
+    variant = "flat",
+    color = "brand",
+    disabled = false,
+    type = "button",
+    size = "medium",
```

Comment:
does storybook still show these default values correctly? I remember a while ago we had to destructure the other way to preserve that

Reply:
That's a nope. Will change

Reply:
Good catch